core-api/src/main/java/org/silverpeas/core/date/Date.java

```diff
@@ -36,7 +35,9 @@
 /**
  * A date on a day of a month and in year. The time isn't represented here. Please refers to
  * DateTime to models also the time.
+ * @deprecated Use the java.time API
  */
+@Deprecated
 public class Date extends AbstractDateTemporal<Date> {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)